kickstart/PXELinux.erb

```diff
@@ -24,6 +24,8 @@ LABEL linux
     APPEND initrd=<%= @initrd %> ks=<%= foreman_url('provision')%> ks.device=bootif network ks.sendmac
     <% elsif @host.operatingsystem.name != 'Fedora' and @host.operatingsystem.major.to_i >= 7 -%>
     APPEND initrd=<%= @initrd %> ks=<%= foreman_url('provision')%> network ks.sendmac
+    <% elsif @host.operatingsystem.name =~ /.*Atomic.*/ -%>
+    APPEND initrd=<%= @host.medium.path %>isolinux/initrd.img repo=<%= @host.medium.path %> ks=<%= foreman_url('provision')%> ks.device=bootif network ks.sendmac
```

Comment:
@domcleal
Following up on your [previous comment](https://github.com/theforeman/community-templates/pull/190#discussion_r32411890), I think I can possibly do without the KERNEL directive and use `vmlinuz` (not tested yet though). 
`initrd` is a bit of a different beast as Red hat in Foreman will try to fetch it from `boot/osfullname-arch/initrd.img`, which isn't Atomic's path.